### DISCUSSION TITLE
On ubuntu with default keybindings you can break your code accidentally?

### DISCUSSION THREAD

**Discussion by zhandosweb** [zed-industries/zed#14461] [Feature Requests]
Hello, friends! I'm Ubuntu user and have installed Zed editor. I liked it, and using it now for Frontend Development. And all the necessary packages have been installed automatically by Zed after it was inspected my project folder, it's cool!

But the default keybindings very strange in my opinion? For example, to open the terminal I need to press CTRL+~ button. And terminal dock from bottom will appear.

And if I press CTRL +` again, it doesn't toggle bottom panel, instead it switches focus between my editor and bottom panel. And to close the bottom panel, I need to first switch focus to the bottom panel, then press. CTRL+J.

But the problem is, if I accidentally switch my focus to the editor, then press CTRL+J, it starting to joint code lines, and it may break your code accidentally. I had to setup autosave before, and my code didn't worked, then realised, that my code lines was jointed, and I forced to load backups from Github. It's strange keybindings behavior in my opinion.

P.s. I'm not a native English speaker, sorry for grammar errors in my text.

**Comment by zhandosweb** [comment]
Also, all the keybindings doesn't work with Russian language, I need to switch my input to English to be able to press any keybinding (like basic CTRL + N to create new tab, etc).

**Reply by tepavcevic** [reply]
Double check which keybindings you are using, since for VSCode (default keybindings) `ctrl-j` works on zed preview and nightly in your described case and also go try to recreate the steps from the linked issue down below to confirm you are having the same issues. 

If that is the case I suspect [this](https://github.com/zed-industries/zed/issues/14366) will fix the issue on stable release which will be out tomorrow.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
